src/machine/asm/execute_aarch64.S

```diff
@@ -41,20 +41,22 @@
 #endif
 
 #define PREPCALL \
-  stp x0, x8, [sp, -96]! SEP \
+  stp x0, x8, [sp, -112]! SEP \
   stp x9, x10, [sp, 16] SEP \
   stp x11, x12, [sp, 32] SEP \
   stp x13, x14, [sp, 48] SEP \
   stp x15, x16, [sp, 64] SEP \
-  stp x17, x18, [sp, 80]
+  stp x17, x18, [sp, 80] SEP \
```

Comment:
If we are not using x18, there's no need to store x18 on the stack. We can simply replace x18 here with x24, which means there's no need to store x25 as well.